### ISSUE TITLE
calving event not detected

### ISSUE THREAD

**Issue by sarahcd** [nilanjanchatterjee/parturition#20]
In our first attempts we got quite good results across a lot of animals. We did notice one case where a calving event is obvious from viewing the Parturition_vel graphs but is not detected:
<img width="607" alt="image" src="https://github.com/nilanjanchatterjee/parturition/assets/15783614/e8f19983-9a45-457f-a395-fc974e7c19bb">

See emails for details to replicate.

**Comment by smthfrmn** [comment]
@nilanjanchatterjee @sarahcd do you know if this is this still an issue?

**Comment by sarahcd** [comment]
Yes, we still don't know the cause of this. I made a workflow instance of just this animal and shared it. 

With the default settings, across 4 years, the calving event is detected 3 times, missed 1 time, and there are 4 false positives. The missed 2017 event is the key result to investigate. For me this would be a high priority!
